### PR TITLE
Capston projects branch

### PR DESCRIPTION
**Pull request by maherramadan78**
I have made my capston project by html and styles css and javascript files  creating the index.html about.html  and styles.css files & about.css.
in the main.js I added the code of java script to make the menu berger work normally, so when it's mobile version , its looks like the menu as three lines , so you click on it and you can see the menu items.
Each of these pages should have versions for 2 different screen sizes:
Mobile: up to 768px wide.
Desktop: 768px or wider.

Links:
The home page should have a link in the menu to the about page.
The logo in the header links to the home page.

Mobile menu:
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.

The mobile menu should have a close (X) button that closes the menu.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+// document.querySelector('.burger-icon-single').addEventListener('click', () => {
+// document.querySelector('.mobile-menu-container').setAttribute('style', 'visibility:hidden');
+// });
+
+// document.querySelector('.burger-icon-single').addEventListener('click', () => {
+// document.querySelector('.mobile-menu-container').setAttribute('style', 'visibility:hidden');
+// });
+
+// Feautre Speaker
```
Comment (uwadonat) [inline main.js]: - [ ] Kindly remove all commented lines to make your project more clean

File: README.md [hunk 2]
```diff
@@ -1,2 +1,90 @@
...
 capston project website
+## html and styles css & javascript
+
+- english language
+- visual code editor
+- html , css, javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://www.loom.com/share/404dfb258692414d9e5bccc10b82129f)
```
Comment (uwadonat) [inline README.md]: - [ ] In your presentation you should highlight a piece of code that you find more interesting or you are proud of making.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,90 @@
 # Capstone-project
 capston project website
+## html and styles css & javascript
+
+- english language
+- visual code editor
+- html , css, javascript
+
+## Live Demo (if available)
+
```
Comment (uwadonat) [inline README.md]: - [ ] It is mandatory in this project to add a link of a deployed application.  You should use github pages to deploy your web pages. Kindly work on it.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,90 @@
 # Capstone-project
 capston project website
+## html and styles css & javascript
+
+- english language
+- visual code editor
```
Comment (uwadonat) [inline README.md]: - Let's follow given [template](https://github.com/microverseinc/readme-template) and customize your readme file.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @maherramadan78 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Descriptive pull request message ✔️
- No linter errors ✔️
- Good usage of github flow ✔️
almost

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (maherramadan78) [reply README.md]: kindly, could you make me remember about deployment application , actually i dont remember

File: README.md (hunk 4 above)
Comment (maherramadan78) [reply README.md]: what should i do over here ? i dont understand

File: main.js (hunk 1 above)
Comment (maherramadan78) [reply main.js]: if i remove the commented line . its going to give me error
so how i fix this kind of issue.?

File: README.md (hunk 4 above)
Comment (uwadonat) [reply README.md]: Just follow the template and add information related to your project like description, links and command to run the project.

File: README.md (hunk 3 above)
Comment (uwadonat) [reply README.md]: - [ ] About deployment, this [link](https://www.codecademy.com/article/f1-u3-github-pages) could be helpful.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,128 @@
+
```
Comment (HAMMAS-SALEEM) [inline about.html:1]: Capture

**[Optional]**

- [ ] As you can see in the image above in your mobile about page you have a white spacing at the top of your page 😨 KIndly remove it to make your project look similar to the design which will make your project look professional 😊

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Hey @maherramadan78 👋
# Status: Approved :green_circle: :1st_place_medal: :100:
success

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Linters
✔️ Following requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 5 above)
Comment (maherramadan78) [reply about.html:1]: ok , thank you i will make it.

**Pull request merged**